microsoft/PowerToys
# Revert "[PowerToys Run] Update to net5 (#12434)"

## Summary of the Pull Request

**What is this about:**
This reverts commit c651a4b36ef104f978a8df294c90f521a2a69e39. We need to update an internal build agent in order to be able to build .net5 projects. For now, we revert those changes because they block the release. 

**What is include in the PR:** 

**How does someone test / validate:** 
- Build an installer and install
- Verify if PowerToys Run works

## Quality Checklist

- [X] **Linked issue:** #12285

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
